src/components/table/table-column-resize/resizable-table-column.component.html

```diff
@@ -6,19 +6,21 @@
     <ng-content select="ux-column-sorting"></ng-content>
 </div>
 
-<div #handle
-     uxDrag
-     uxFocusIndicator
-     tabindex="0"
-     aria-label="Column resize handle. Use arrow keys to change the column width."
-     class="ux-resizable-table-column-handle"
-     *ngIf="!disabled"
-     (onDragStart)="onDragStart($event)"
-     (onDrag)="onDragMove($event, handle)"
-     (onDragEnd)="onDragEnd()"
-     (keydown.ArrowLeft)="onMoveLeft()"
-     (keydown.ArrowRight)="onMoveRight()"
-     (click)="$event.stopPropagation()">
+<ng-container *ngIf="!hideHandle">
```

Comment:
Is there a problem if you add this condition to the handle element directly? E.g.
```html
<div #handle *ngIf="!disabled && !hideHandle" ...>
```